src/app/endround/endround.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, Input, SimpleChanges } from '@angular/core';
+
+@Component({
+  selector: 'app-endround',
+  templateUrl: './endround.component.html',
+  styleUrls: ['./endround.component.scss'],
+})
+
+export class EndroundComponent {
+  @Input() match!: any;
+  teamWon = 0;
+  ngOnChanges(changes: SimpleChanges) {
```

Comment:
Like eslint suggests, EndroundComponent should implement OnChanges if you use ngOnChanges

Reply:
resolved in 23bb25d